static/js/main.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-var */
+// Notice: IE is still supported, so disable eslint for the file
```

Comment:
Probably worth being specific about what version we support:

```suggestion
// Notice: IE 11 is still supported, so disable eslint for the file
```

Reply:
`const` is supported in IE11, as mentioned in https://github.com/nodejs/nodejs.org/pull/3540, it's only <IE11 that even has an issue with this